package.json

```diff
@@ -2,20 +2,40 @@
   "name": "@cheqd/mcp",
```

Comment:
See discussion in Slack

Reply:
ideally it should just be cheqd. that's the convention.

Should we have it as cheqd or @cheqd/mcp-toolkit. creds-libs uses cheqd